src/components/Calculator.css

```diff
@@ -0,0 +1,72 @@
+body {
+  background-color: #fff;
+}
+
+.app {
+  display: flex;
+  min-height: 80vh;
+  align-items: center;
+  justify-content: center;
+}
+
+.calculator {
+  width: 100%;
+  max-width: 500px;
+  background-color: white;
+  border-radius: 4px;
+  overflow: hidden;
+  box-shadow: 0 2px 10px rgba(0, 0, 0, 0.3);
+}
+
+.display {
+  padding: 20px;
+  text-align: right;
+  background-color: #888;
+  color: #fff;
+  font-size: 20px;
+  font-weight: 500;
+}
+
+.display span {
+  font-size: 20px;
+  color: #fff;
+}
+
+button {
+  appearance: none;
+  border: none;
+  outline: none;
+  color: #e6e6e6;
+  font-size: 20px;
+  padding: 16px;
+  cursor: pointer;
+  transition: 0.2s;
+}
+
+button:hover {
+  opacity: 0.5;
+}
```

Comment:
- [ ] It would be nice if you could match your design with the one given in the project requirements. Currently, your design for the calculator app does not match the original design. Please modify your app design to have all buttons and signs used in this calculator project. 👍🏽

**The Project Requirements**

<img width="818" alt="Screenshot 2023-01-13 at 9 47 58 AM" src="https://user-images.githubusercontent.com/50941074/212290825-d30c80bb-cb5d-4d38-bd8e-99400e2512b6.png">

**Your Design**

<img width="524" alt="Screenshot 2023-01-13 at 9 47 08 AM" src="https://user-images.githubusercontent.com/50941074/212290885-8296e33f-1de3-4205-aba7-0412eef5a957.png">

- [ ] Currently, your design does not have the number 1, and counting does not follow the numbering flow of the original design. Please work on them 👍🏽